openedx_rocketchat/utils.py

```diff
@@ -0,0 +1,205 @@
+import hashlib
+import logging
+import re
+
+from django.conf import settings
+from django.core.cache import cache
+
+from openedx_rocketchat.edxapp_wrapper.get_configuration_helpers import get_configuration_helpers
+from openedx_rocketchat.edxapp_wrapper.get_student_models import anonymous_id_for_user, get_user
+
+from rocketchat_API.rocketchat import RocketChat as ApiRocketChat
+from rocketchat_API.APIExceptions.RocketExceptions import RocketAuthenticationException, RocketConnectionException
+
+
+LOG = logging.getLogger(__name__)
+CACHE_TIMEOUT = get_configuration_helpers().get_value("ROCKET_CHAT_CACHE_TIMEOUT", 86400)
```

Comment:
Is this a global setting?, if so, this could be inside of the settings plugins and then get it by django.conf instead of configuration helpers. @andrey-canon

Reply:
That is not microsite aware

Reply:
Ok.